.gitignore

```diff
@@ -17,3 +17,6 @@ doc/
 
 # Ignore Mkdocs files
 site/
+
+.bundle/
```

Comment:
What is the `.bundle` directory?

Reply:
Where the config file is stored.  Or, the file that holds `bundle config --local` operations.